app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -334,6 +353,22 @@ protected void setMetaData(StatusViewData.Concrete statusViewData, StatusDisplay
             timestampText = metaInfo.getContext().getString(R.string.post_timestamp_with_edited_indicator, timestampText);
         }
         metaInfo.setText(timestampText);
+
+        metaInfo.setBackgroundColor(getTimeBasedColor(now - then));
```

Comment:
This is also used in the notifications fragment.

Not sure if it is desired there.